GEOSldas_App/lenkf_j_template.py

```diff
@@ -63,7 +63,11 @@
 
 source $GEOSBIN/g5_modules
 
-# OPENMPI flags 
+setenv MPI_STACK {DETECTED_MPI_STACK}
+
+if ( ${{MPI_STACK}} == "openmpi" ) then
```

Comment:
Are we missing indents in this if-else block (and in a couple of other places in the script)?  Not sure if they're required for py, but they would certainly help mere humans such as myself.